.drone.yml

```diff
@@ -27,6 +27,7 @@ steps:
   - npm publish packages/datetime
   - npm publish packages/table
   - npm publish packages/select
+  - npm publish packages/popover2
```

Comment:
When you change a `.drone.yml` you need call @militarpancho to sign the new drone.